conf/local.env

```diff
@@ -1,5 +1,5 @@
 GIN_MODE=release
-LOG_LEVEL=DEBUG
+LOG_LEVEL=INFO
```

Comment:
I would keep local.env and test.env in debug level. It's for developement so it's better to see more here.